CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `DataArray.to_hdf5()` accepts both file handles and file paths.
 
 ### Fixed
+- Add enable_caching parameter for `run()`, allow users to override the global cache setting for individual task submissions.
```

Comment:
please update the changelog item so users can see how to set this. For example
- Users can toggle task caching through `from tidy3d.web.core.environment import Env`. `Env.set_caching(enableCaching: bool = True)`.

Also, this should probably be in the `### Added` section.

Reply:
modified